sci-geosciences/qmapshack/qmapshack-1.17.1_p20250426.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake xdg
+
+COMMIT="948ae8a8a1526766ebdf9df2f207462c09c15c3b"
+
+DESCRIPTION="GPS mapping utility"
+HOMEPAGE="https://github.com/Maproom/qmapshack/wiki"
+SRC_URI="https://github.com/Maproom/${PN}/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}"/${PN}-${COMMIT}
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="dbus"
+
+RDEPEND="
+	dev-db/sqlite
+	>=dev-libs/quazip-1.3:0=[qt6]
```

Comment:
Should be `>=dev-libs/quazip-1.3-r2:=[qt6(+)]` - reasons:
- 1.3-r2 introduced IUSE qt6
- any future quazip bump will drop IUSE qt5, qt6 and make it Qt6-based unconditionally
- that would necessitate fixing any revdep requiring quazip[qt6] without defining USEdep-default in case the flag is missing